frontend/src/Store/Actions/movieIndexActions.js

```diff
@@ -223,16 +223,28 @@ export const defaultState = {
       type: filterBuilderTypes.EXACT,
       valueType: filterBuilderValueTypes.BOOL
     },
+    {
+      name: 'isAvailable',
+      label: translate('Is Available'),
+      type: filterBuilderTypes.EXACT,
+      valueType: filterBuilderValueTypes.BOOL
+    },
+    {
+      name: 'minimumAvailability',
+      label: translate('Minimum Availability'),
+      type: filterBuilderTypes.EXACT,
+      valueType: filterBuilderValueTypes.MINIMUM_AVAILABILITY
+    },
     {
       name: 'title',
       label: translate('Title'),
       type: filterBuilderTypes.STRING
     },
     {
       name: 'status',
-      label: translate('Status'),
+      label: translate('Release Status'),
```

Comment:
Translate keys wont have spaces in them

`src\NzbDrone.Core\Localization\Core\en.json` is where you would check for existing keys or add new ones as needed